sdk/communication/azure-communication-phonenumbers/CHANGELOG.md

```diff
@@ -1,14 +1,18 @@
 # Release History
 
-## 1.2.0-beta.1 (Unreleased)
+## 1.2.0-beta.2 (2024-03-01)
 
 ### Features Added
+- Add support for number lookup
+  - Format only can be returned for no cost
+  - Additional number details can be returned for a cost
 
-### Breaking Changes
+## 1.2.0-beta.1 (2023-09-07)
```

Comment:
This looks out of order. We have releases in December 2023 below this. These should be sorted in reverse chronological order.